bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperUtil.java

```diff
@@ -47,6 +47,12 @@
 import static org.junit.Assert.*;
 
 public class ZooKeeperUtil {
+
+    static {
```

Comment:
@sijie for 3.5.3 most of four letter words commands are disabled by default in order to prevent DoS attacks. We are using ClientBase to wait for ZK server to start and this uses the fourbletter words API.
Without this change  tests will be very slow because the wait will always timeout